### PR TITLE
pe-bear: init at 0.6.1

### PR DESCRIPTION
**Pull request by blitz**
###### Description of changes

PE-Bear is a PE binary analysis tool. It allows open PE files and "surfing" the binary.

See: https://hshrzd.wordpress.com/pe-bear/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RaitoBezarius** [comment]
Result of `nixpkgs-review pr 203139` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pe-bear</li>
  </ul>
</details>

File: pkgs/applications/misc/pe-bear/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  ];
+
+  buildInputs = [
+    qtbase
+  ];
+
+  meta = with lib; {
+    description = "Portable Executable reversing tool with a friendly GUI";
+    homepage = "https://hshrzd.wordpress.com/pe-bear/";
+    license = licenses.gpl2Only;
```
Comment (RaitoBezarius) [inline pkgs/applications/misc/pe-bear/default.nix:33]: `pe-bear` is apparently based on capstone & https://github.com/hasherezade/bearparser/blob/master/LICENSE
as those are vendored dependencies, I think this should be `[ (pe-bear license) (capstone license) (bearparser license) ]` but I don't recall if there is a precedent for that.

**Comment by tfc** [comment]
It would be great if we had some `drv.passthru.tests.some-basic-check` test for this in order to make maintenance easier.

**Comment by RaitoBezarius** [comment]
> It would be great if we had some `drv.passthru.tests.some-basic-check` test for this in order to make maintenance easier.

Uncertain if it's easy to test the actual features of this GUI application, but `passthru.tests = testers.testVersion { package = pe-bear; }` could be a great start if `PEbear --version` works fine?

**Comment by blitz** [comment]
> > It would be great if we had some `drv.passthru.tests.some-basic-check` test for this in order to make maintenance easier.
> 
> Uncertain if it's easy to test the actual features of this GUI application, but `passthru.tests = testers.testVersion { package = pe-bear; }` could be a great start if `PEbear --version` works fine?

Unfortunately, the app interprets any argument as the PE file to open and has no command line parsing at all. So I can't even test `--version`. :(

File: pkgs/applications/misc/pe-bear/default.nix (hunk 1 above)
Comment (blitz) [reply pkgs/applications/misc/pe-bear/default.nix]: I've added the licenses of the vendored libs.

**Pull request merged**